src/components/TypeComponents/SimpleSearch/Form/filters/SimpleSearchDateFilterField.js

```diff
@@ -29,7 +29,14 @@ const SimpleSearchDateFilterField = ({
   const { values } = useFormState();
   const { change } = useForm();
 
-  const isSetOrUnset = get(values, `${name}.comparator`) === 'isNull' || get(values, `${name}.comparator`) === 'isNotNull';
+  const comp = values?.[name?.comparator];
```

Comment:
Following the suggestion below you can have this is as 
```const comparatorPresent = isComparatorPresent(values?.[name?.comparator])```

and replace the usage of `comparatorSufficient` throughout this file and the other files.

You would have to ofcourse import the function from the utilities folder in each of these files.

At the same time you can remove the `lodash get` import as we arent using it anymore.